src/TritonCTS/README.md

```diff
@@ -49,7 +49,7 @@ If this parameter is omitted, the code gets the value from ten times the height
 - ``clk_nets`` (optional) is a string containing the names of the clock roots. 
 If this parameter is omitted, TritonCTS looks for the clock roots automatically.
 - ``out_path`` (optional) is the output path (full) that the lut.txt and sol_list.txt files will be saved. This is used to load an existing characterization, without creating one from scratch.
-- ``only_characterization`` (optional), if true, makes so that the code exits after running the characterization.
+- ``characterization_only`` (optional), if true, makes so that the code exits after running and checking the characterization.
```

Comment:
Could use some rewording. Flags are not "true" or false.
How about
When the `-characterization_only` option is specified the only the library characterization step is run and not clock tree is inserted in the design.